src/hotspot/share/code/compressedStream.hpp

```diff
@@ -111,4 +112,93 @@ class CompressedWriteStream : public CompressedStream {
   }
 };
 
+class CompressedBitStream : public ResourceObj {
```

Comment:
Maybe it is better `CompressedSparseData`?

Reply:
OK